dsc/tests/dsc_settings.tests.ps1

```diff
@@ -0,0 +1,102 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+Describe 'tests for dsc settings' {
+    BeforeAll {
+
+        $script:policyFilePath = if ($IsWindows) {
+            Join-Path $env:ProgramData "dsc" "settings.dsc.json"
+        } else {
+            "/etc/.dsc/settings.dsc.json"
```

Comment:
Is the app folder generally hidden as `.dsc` or should it just be `dsc`?

Reply:
Good point.
For settings, it probably should not be hidden.

Changed location to "/etc/dsc/settings.dsc.json" throughout the code and updated PR description.